CONFIGURATION.md

```diff
@@ -96,6 +96,51 @@ Add the following and change state and params to the page you want:
         "state": "public.posts"
         "params": {}
 ```
+### How to integrate admobs
+step 1: Run this command in your project directory
+	```cordova plugin add cordova-plugin-admob```
```

Comment:
If admob does not work, try admobpro `cordova plugin add cordova-plugin-admobpro`